spyder/config/main.py

```diff
@@ -432,6 +432,8 @@
               'profiler/run profiler': "F10",
               # ---- In widgets/ipythonconsole/shell.py ----
               'ipython_console/new tab': "Ctrl+T",
+              'ipython_console/new pylab tab': "Ctrl+U",

```

Comment:
Is there a particular motivation for the choice of these shortcuts? Is it the best idea to occupy top-level (Ctrl + Letter) shortcuts with these commands?

Reply:
The only motivation is `T` `Y` `U` are next to each other on the keyboard and `S` and `P` were already taken.  I don't really need any of those shortcuts that often even "Ctrl+T".

Reply:
I think the IPython console uses Emacs-style shortcuts, so Ctrl-U means "kill everything in front of the cursor" and Ctrl-Y means "yank previously killed text" (kill and yank are the Emacs terms for cut and paste). Maybe it's best not to introduce the shortcuts, at least unless somebody actually wants them?

Reply:
> Maybe it's best not to introduce the shortcuts

I agree. I'll remove the shortcuts defaults.  I'll try leave them as unset in the shortcut preferences so users can set them to whatever they.

Reply:
Sounds good to me!

Reply:
I can't seem to get the shortcuts to work at all no matter what I set them to.  I must be missing a place where I need to register them. So these shortcuts don't currently work at all.

If I set them to `None` they appear blank in the preferences and I can set them. But the shortcuts conflict warning comes up saying they are set to `None` and they wouldn't work when I set them.

This is outside the scope of this PR so the shortcuts were removed entirely